cmake/stm32/mp1.cmake

```diff
@@ -0,0 +1,26 @@
+set(STM32_MP1_TYPES 
+    MP151Axx MP151Cxx MP151Dxx MP151Fxx
+    MP153Axx MP153Cxx MP153Dxx MP153Fxx 
+    MP157Axx MP157Cxx MP157Dxx MP157Fxx)
+

```

Comment:
look at F3 family. If I remember correctly this must match the existing startup scripts

Reply:
That would mean that a STM32F157F chip would be of STM32C157 type? Is this really what we want?

Reply:
from my understanding yes this is what we want. The "type" corresponds to linker script type. Type reflects the peripherals only. While same chip exists with different size and package it doesn't impact the type

Reply:
Ok!